spec/models/verse_spec.rb

```diff
@@ -58,4 +58,12 @@
       )
     end
   end
+
+  describe 'verse of the day' do
+    it 'returns the correct verse' do
```

Comment:
I can get this to fail pretty readily. The reason it fails is because the test assumes you only have 100 Verses, but if other tests run first it increments the sequence before you create the list and the Verse day starts on 7 or 8.

Reply:
Okay that makes sense. Probably going to test it by creating a single verse first if that will work.

Reply:
This works. It looks to me like we'll never hit Verse #1, since we always add 1 to the current day. This test fails:
```ruby
describe '.verse_of_the_day' do
  include ActiveSupport::Testing::TimeHelpers

  it 'returns the correct verse' do
    # year, month, day, hour, minute, second
    travel_to Time.local(2024, 1, 1, 1, 0, 0) do
      verse = create(:verse, day: 1)
      expect(Verse.verse_of_the_day.scripture).to eq(verse.scripture)
    end
  end
end
```

Reply:
Ahh got it. Let me update my example and I'll be right back.

Reply:
```ruby
it 'returns the correct verse' do
  # year, month, day, hour, minute, second
  travel_to Time.local(2024, 4, 9, 1, 0, 0) do
    verse = create(:verse, day: 1)

    expect(Verse.verse_of_the_day.scripture).to eq(verse.scripture)
  end

  travel_to Time.local(2024, 4, 10, 1, 0, 0) do
    verse = create(:verse, day: 2)

    expect(Verse.verse_of_the_day.scripture).to eq(verse.scripture)
  end
end
```

Reply:
Okay awesome! Makes perfect sense!! Thanks so much for your help! 🙏